tests/MenderAPI/deviceconnect.py

```diff
@@ -0,0 +1,50 @@
+# Copyright 2020 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        https://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+
+import pytest
+import ssl
+
+from testutils.util import websockets
+from . import api_version
+from . import get_container_manager
+
+
+class DeviceConnect:
+    def __init__(self, auth, devauth):
+        self.reset()
+        self.auth = auth
+        self.devauth = devauth
+
+    def reset(self):
+        # Reset all temporary values.
+        pass
+
+    def get_websocket_url(self):
```

Comment:
I suggest marking this as private
```suggestion
    def _get_websocket_url(self):
```

Reply:
I disagree. This is available to anyone who wants it, and the other modules in the same directory are having similar public members.